packages/orbit-components/src/ListChoice/index.tsx

```diff
@@ -108,32 +32,52 @@ const ListChoice = React.forwardRef<HTMLDivElement, Props>(
     };
 
     return (
-      <StyledListChoice
+      <div
+        className={cx(
+          "orbit-list-choice",
+          "py-sm px-md box-border flex w-full items-center",
+          "border-b-cloud-normal bg-white-normal border-b-solid border-b",
+          "duration-fast transition-[background-color] ease-in-out",
+          disabled ? "cursor-not-allowed" : "cursor-pointer",
+          "hover:outline-none [&:hover_button]:bg-transparent",
+          !disabled && "hover:bg-cloud-light group",
+          "[&_.orbit-checkbox-styled-label]:w-auto",
+        )}
         onClick={!disabled ? onClick : undefined}
         data-test={dataTest}
         id={id}
         ref={ref}
         onKeyDown={!disabled ? handleKeyDown<HTMLDivElement>(onClick) : undefined}
         tabIndex={tabIndex || disabled ? -1 : 0}
-        disabled={disabled}
         data-title={title}
         aria-disabled={disabled}
         aria-selected={selected}
         role={role || (selectable ? "checkbox" : "button")}
         {...conditionalProps}
       >
-        {icon && <StyledListChoiceIcon>{icon}</StyledListChoiceIcon>}
-        <StyledListChoiceContent>
+        {icon && (
+          <div
+            className={cx(
+              "group-hover:[&_svg]:text-icon-primary-foreground",
```

Comment:
Just realised this has no effect. `theme.orbit.colorIconPrimary` is used both for the icon color and the icon color on hover. @DSil could you confirm this?

Reply:
That's true. Because we are overriding the icon's color no matter what with the class `[&_svg]:text-icon-primary-foreground`, I'd say this is not needed.